docs/en/latest/FAQ.md

```diff
@@ -21,74 +21,80 @@ title: FAQ
 #
 -->
 
-## Why a new API gateway?
+## Why do I need a new API gateway?
 
-There are new requirements for API gateways in the field of microservices: higher flexibility, higher performance requirements, and cloud native.
+As organizations move towards cloud native microservices, there is a need for an API gateway that is performant, flexible, secure and scalable.
 
-## What are the differences between Apache APISIX and other API gateways?
+APISIX outperforms other API gateways in these metrics while delivering functions like dynamic routing, a plugin mechanism, a low code dashboard and more.
 
-Apache APISIX is based on etcd to save and synchronize configuration, not relational databases such as Postgres or MySQL.
+## How does Apache APISIX differ from other API gateways?
 
-This not only eliminates polling, makes the code more concise, but also makes configuration synchronization more real-time. At the same time, there will be no single point in the system, which is more usable.
+Apache APISIX differs in the following ways:
 
-In addition, Apache APISIX has dynamic routing and hot loading of plug-ins, which is especially suitable for API management under micro-service system.
+- It uses etcd to save and synchronize configurations rather than relational databases like PostgreSQL or MySQL. This eliminates the need for polling, synchronizes the configuration in real-time, makes the code concise and avoids a single point of failure.
+- Supports dynamic routing.
+- Supports [hot loading of Plugins](/docs/apisix/plugins/#hot-reload).
 
-## What's the performance of Apache APISIX?
+## What is the performance impact of using Apache APISIX?
 
-One of the goals of Apache APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](benchmark.md)
+Apache APISIX delivers the best performance among other API gateways with a single-core QPS of 23,000 with an average delay of 0.6 ms.
```

Comment:
Could you double check on the QPS and the latency stats? I remember the stats are 18,000 QPS and 0.2 ms latency on the [GitHub intro page](https://github.com/apache/apisix).

![image](https://user-images.githubusercontent.com/36651058/159835483-3f09c5e9-d6b6-4470-881d-0d0333005f0f.png)

Reply:
I mean either of them or all of them could be right. But we need to keep stats correct.